### PR TITLE
[PM-7172] Create account security settings component (navigational changes)

### PR DESCRIPTION
**Pull request by djsmith85**
## Type of change

```
- [ ] Bug fix
- [X] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Split out the account security related controls into a separate page according to the UI re-design (navigational changes)

This only contains navigational changes and makes it easier to enable code-ownership
`settings.component` and `about.component` are now under tools-ownership
and
`account-security.component` is under auth-ownership

The visual changes planned for the UI refresh will be done, once all needed base components from the Component Library are available. 

## Code changes

- **Move about.component into tools ownership-** https://github.com/bitwarden/clients/commit/ae15023ee01bb95d34f5935050c0ce204560d252
  - Move `about.component` to `tools/popout/settings/about/`
- **Split out account security settings https://github.com/bitwarden/clients/commit/83c4637f5ea21fc619feab63123757b2805942e3** 
  - Move `settings.component.ts` to `auth/popup/settings` and rename to `account-security.component.ts`
  - Move controls from `settings.component.html` and create `account-security.component.html`
  - Move `settings.component.html` to `tools/popup/settings.component.html`
  - Create `settings.component.ts` under `tools/popup/settings`
  - Fixup module imports and routing
  - Add new strings to `en/message.json`
- **Move vault-timeout-input.component to auth -** https://github.com/bitwarden/clients/pull/8817/commits/b83b038108391e255a795efb162c1e06ca83e704
- **Move await-desktop-dialog.component to auth -** https://github.com/bitwarden/clients/pull/8817/commits/eb039462898504ba347178680a0286c4801c480b
- **Add transition for account-security -** https://github.com/bitwarden/clients/pull/8817/commits/362fb8e0e8af49ce25a7c9534aa59cf273783010
### Before
before_settings
before_settings2

### After
after_settings
after_account-security

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by jlf0dev — APPROVED** [review]
Looks good for Auth!

**Comment by jlf0dev** [comment]
got some tests failing

File: apps/browser/src/auth/popup/settings/account-security.component.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  </div>
+  <div class="box list">
+    <h2 class="box-header">{{ "sessionTimeoutHeader" | i18n }}</h2>
+    <div class="box-content single-line">
+      <app-callout type="info" *ngIf="vaultTimeoutPolicyCallout | async as policy">
+        <span *ngIf="policy.timeout && policy.action">
+          {{
+            "vaultTimeoutPolicyWithActionInEffect"
+              | i18n: policy.timeout.hours : policy.timeout.minutes : (policy.action | i18n)
+          }}
```
Comment (audrey-jensen) [inline apps/browser/src/auth/popup/settings/account-security.component.html:50]: 💭 I wonder whether there's a way that the format string could look up the hours and minutes values on its own, similar to a .net formatter. Same goes for figuring out a way to apply the i18n pipe to `policy.action`

File: apps/browser/src/tools/popup/settings/settings.component.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  [DeviceType.FirefoxExtension]:
+    "https://addons.mozilla.org/en-US/firefox/addon/bitwarden-password-manager/#reviews",
+  [DeviceType.OperaExtension]:
+    "https://addons.opera.com/en/extensions/details/bitwarden-free-password-manager/#feedback-container",
+  [DeviceType.EdgeExtension]:
+    "https://microsoftedge.microsoft.com/addons/detail/jbkfoedolllekgbhcbcoahefnbanhhlh",
+  [DeviceType.VivaldiExtension]:
+    "https://chromewebstore.google.com/detail/bitwarden-free-password-m/nngceckbapebfimnlniiiahkandclblb/reviews",
+  [DeviceType.SafariExtension]: "https://apps.apple.com/app/bitwarden/id1352778147",
+};
```
Comment (audrey-jensen) [inline apps/browser/src/tools/popup/settings/settings.component.ts:30]: ⛏️ Seems like this data should be encapsulated by a service and/or observable. The observable would encapsulate the lookup, then all this page would need to do is open the popup and feed it the observeable.

**Review by audrey-jensen — APPROVED** [review]
⛏️ nitpick
💭 idle thoughts

**Review by jlf0dev — APPROVED** [review]

File: apps/browser/src/popup/app-routing.module.ts [hunk 3]
```diff
@@ -246,6 +247,12 @@ const routes: Routes = [
     canActivate: [AuthGuard],
     data: { state: "autofill" },
   },
+  {
+    path: "account-security",
```
Comment (audrey-jensen) [inline apps/browser/src/popup/app-routing.module.ts:251]: ⛏️ Similar to the other settings factoring PRs, I'm wondering whether we should call out that this is a setting screen in the path.

**Review by audrey-jensen — APPROVED** [review]
⛏️ nitpick

**Review by jlf0dev — APPROVED** [review]

**Pull request merged**